dotnet/src/SemanticKernel/CoreSkills/ConditionalSkill.cs

```diff
@@ -0,0 +1,316 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Xml;
+using Microsoft.SemanticKernel.AI;
+using Microsoft.SemanticKernel.KernelExtensions;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.Planning;
+using Microsoft.SemanticKernel.SkillDefinition;
+
+namespace Microsoft.SemanticKernel.CoreSkills;
+public class ConditionalSkill
+{
+    #region Prompts
+    internal const string IfStructureCheckPrompt =
```

Comment:
Maybe out of scope of this PR, but I'm wondering, can we store core skill prompts in separate txt files and read from them? The same applies for Planner and other skills. In this way we can keep our C# classes nice and clean.

Reply:
Sure, I think this could be a improvement to tackle